scripts/pylib/twister/twisterlib/harness.py

```diff
@@ -419,11 +419,11 @@ def generate_command(self):
             for fixture in handler.options.fixture:
                 command.append(f'--twister-fixture={fixture}')
 
+        command.extend(pytest_args_yaml)
```

Comment:
well, now it should work as expected:
https://github.com/zephyrproject-rtos/zephyr/blob/4c16ec1fba56c5631fb0ab19583a8a30d7ded529/doc/develop/test/pytest.rst?plain=1#L100-L101

Reply:
would be nice to adjust the appropriate test to be more specific on arguments' precedence:

https://github.com/zephyrproject-rtos/zephyr/blob/4c16ec1fba56c5631fb0ab19583a8a30d7ded529/scripts/tests/twister/pytest_integration/test_harness_pytest.py#L83-L85

Reply:
test updated, thank you!